recipes/schnetpack/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "schnetpack" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a3d590cd92c32324dc3839bd1055a02a90d20092b81937cc13f92c5c5c3ded7a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - pytorch-cpu >=0.4.1
+    - numpy
+    - ase >=3.18
+    - h5py
+    - tensorboardX
+    - tqdm
+    - pyyaml
+    - pytest-console-scripts
```

Comment:
is pytest really a runtime dependency?

Reply:
It is just used for testing I guess, but unfortunately it is listed in `install_requires`

Reply:
I opened a PR upstream https://github.com/atomistic-machine-learning/schnetpack/pull/164